test/functionalTest/cases/3122_location_metadata_issues/get_location_attr_with_db_creation.test

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This test covers:

![imagen](https://user-images.githubusercontent.com/1534240/48832632-e2669700-ed79-11e8-8670-b14c6a231829.png)

Great!

NTC

Reply:
It's ok